partiql-cli/src/test/kotlin/org/partiql/cli/functions/ReadFileTest.kt

```diff
@@ -84,7 +85,8 @@ class ReadFileTest {
         val path = getResourcePath("data.ion")
         val args = listOf("\"$path\"").map { it.exprValue() }
         assertThrows<IllegalStateException> {
-            function.callWithOptional(session, args, "{type:\"ion\"}".exprValue())
+            function.callWithRequired(session, args)
```

Comment:
I'm not sure which of these throws the `IllegalStateException`. If they both do, can you make 2 separate tests? Or assert these sequentially?